### ISSUE TITLE
Open MeetingSession.audioVideoController for extension

### ISSUE THREAD

**Issue by jamesnakagawa** [aws/amazon-chime-sdk-js#1860]
# Tell us about your request

Hello, I'm from [VRIFY Technologies](https://www.vrify.com), we use your service as an essential component of our product. I'm one of the web developers on the team.

I'm trying to improve connectivity in our video conferencing and manage bandwidth. We enabled the simulcast feature and it seems to be an improvement.

However, in trying to improve things further, I'm hindered by a lack of access to the AudioVideoController held by MeetingSession. It seems like access here is required to restart video feeds and enact refreshes of the uplink/downlink policies. (If there's a better way, please let me know!)

Specifically, the methods I'd like to access are `update` and `updateLocalVideoFromPolicy`.

My actual request is to change `MeetingSession.audioVideoController` from `private` to `protected`, so that I can safely extend the class and add my own functionality to it. Of course, I'm open to suggestion for my approach or proposed solution.

**Comment by jamesnakagawa** [comment]
I've opened a PR to do this at #1861.

**Comment by hensmi-amazon** [comment]
Hi James, can you expand on what you mean by 
* 'required to restart video feeds': Is this in reference to local or remote video?
* 'enact refreshes': We re-check the policies every two seconds, is this delay what you would like to reduce?

**Comment by jamesnakagawa** [comment]
Hi Henry, yes of course. Thanks for your fast reply. We've actually had two use-cases where this change is relevant.

1. I was referring to local video here. We have an issue where videos are freezing (not turning off) for certain users and we're unsure of the cause. However, calling `update` on the AudioVideoController fixes things. I have retained the logs and can detail a bug report if you would like.
2. I didn't notice the refresh after 2 seconds as you say. Let me try again and see if it is still needed.

**Comment by jamesnakagawa** [comment]
A quick update:

It turns out I don't need to manually re-check the policy, but it seems like I need to restart the local video feed after the policy has been updated. Since this happens every 2 seconds, I can call `meetingSession.audioVideoController.update` on a timer.

**Comment by hensmi-amazon** [comment]
That sounds like a bug, so I'd rather fix that then call `update` every 2 seconds, as it can occasionally be an expensive operation.  Can you submit a report?

**Comment by jamesnakagawa** [comment]
Absolutely, I'll put one together.

**Comment by devalevenkatesh** [comment]
@jamesnakagawa I am closing this issue due to no-activity but please feel free to re-open this issue or a new one with a bug template so that we can get the necessary information such as logs, browser version and so on for any further help / debugging. Thanks for reporting this issue.
